chain/network/src/peer.rs

```diff
@@ -850,40 +853,40 @@ impl StreamHandler<Result<Vec<u8>, ReasonForBan>> for Peer {
                 };
                 self.chain_info = handshake.chain_info.clone();
                 self.peer_manager_addr
-                    .send(Consolidate {
-                        actor: ctx.address(),
-                        peer_info: peer_info.clone(),
-                        peer_type: self.peer_type,
-                        chain_info: handshake.chain_info.clone(),
-                        this_edge_info: self.edge_info.clone(),
-                        other_edge_info: handshake.edge_info.clone(),
-                    })
-                    .into_actor(self)
-                    .then(move |res, act, ctx| {
-                        match res {
-                            Ok(ConsolidateResponse::Accept(edge_info)) => {
-                                act.peer_info = Some(peer_info).into();
-                                act.peer_status = PeerStatus::Ready;
-                                // Respond to handshake if it's inbound and connection was consolidated.
-                                if act.peer_type == PeerType::Inbound {
-                                    act.edge_info = edge_info;
-                                    act.send_handshake(ctx);
+                        .send(Consolidate {
```

Comment:
seems like just formatting change. Did you run `cargo fmt --all` (I recommend running formatter automatically on save)?

Reply:
I'm running formatted automatically on save. I also ran cargo fmt --all to make sure.